### PR TITLE
docs: add Gitpod quickstart to blog tutorial

### PR DESCRIPTION
**Pull request by jacobPARIS**
This PR adds a Gitpod quickstart to the blog tutorial. It is currently linked to my fork of `indie-stack` that has Gitpod configured (PR for that at https://github.com/remix-run/indie-stack/pull/58)

The goal is for people following the blog tutorial to not have to worry about setting up their local environment or installing Fly. In one click, they should be able to get into a code editor with the demo application running.
 
Message from Kent 'Could you open a PR? I want to see what the experience would be like on the blog tutorial'

This will remain a draft until the workflow is deemed good enough to merge into Indie Stack, at which point I will update this badge to point at the base repository and remove the Draft tag

## Try it out

Click this button to create a [Gitpod](https://gitpod.io) workspace with the project set up and ready to run in VS Code or JetBrains either directly in the browser or on the desktop.

[Gitpod Ready-to-Code](https://gitpod.io/#https://github.com/jacobPARIS/indie-stack/)

## Demo (opening in local VS Code)

https://user-images.githubusercontent.com/5633704/162593724-72534e69-9ce3-4e72-bf1b-ac7aadd1eea5.mp4

## Demo (opening in browser)

https://user-images.githubusercontent.com/5633704/162593864-37689321-2abf-4c73-9bde-7b63348eb52a.mp4

### CODE REVIEW

File: docs/tutorials/blog.md [hunk 1]
```diff
@@ -11,14 +11,20 @@ We're going to be short on words and quick on code in this quickstart. If you're
 
 This tutorial uses TypeScript. Remix can definitely be used without TypeScript. We feel most productive when writing TypeScript, but if you'd prefer to skip the TypeScript syntax, feel free to write your code in JavaScript.
 
+
 ## Prerequisites
 
+Click this button to create a [Gitpod](https://gitpod.io) workspace with the project set up and ready to run in VS Code or JetBrains either directly in the browser or on the desktop.
+
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/jacobPARIS/indie-stack/)
```
Comment (kentcdodds) [inline docs/tutorials/blog.md:19]:
```suggestion
[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/remix-run/indie-stack)
```

Once this is merged: https://github.com/remix-run/indie-stack/pull/58

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Yeah, this is neat 👍

**Comment by andrewbrey** [comment]
@jacobPARIS I just gave this a try against your repository branch, and noted the prisma dev migration is not done automatically at container init. That might be a nice thing to add to the setup script (like you're doing when you create the `.env` file for the user). If you did that, clicking "open in gitpod" would launch into a fully working and initialized dev environment, db and all :+1:

**Comment by jacobPARIS** [comment]
> @jacobPARIS the prisma dev migration is not done automatically at container init

Good catch, that is definitely worth including

**Review by kentcdodds — APPROVED** [review]

**Pull request merged**